packages/button/src/Button.js

```diff
@@ -147,6 +147,7 @@ const buttonStyles = (props, baseTheme) => {
     cursor: props.disabled ? 'default' : 'pointer',
     fontWeight: theme.Button_fontWeight,
     height: theme[`Button_size_${props.size}`],
+    lineHeight: 0.75, // theme.fontSize_base / parseFloat(theme.ButtonContent_fontSize_small)
```

Comment:
We should probably include the actual calculation here and also do some experimentation/validation that it works across various values.